app/views/contacts/show.html.erb

```diff
@@ -0,0 +1,32 @@
+<% content_for(:navigation) { back_link(path: :back) } %>
+<% content_for(:page_title) { t('.h1-heading') } %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl"><%= content_for(:page_title) %></h1>
+
+    <p class="govuk-body"><%= t('.page_header') %></p>
+
+    <p class="govuk-body">
+      <%= t('.case_inquiries.header') %>
+      <br />
```

Comment:
I think we should be using GOV-UK spacing elements rather than `br` tags in our markup. See https://design-system.service.gov.uk/styles/spacing/

Reply:
But according to the prototype https://legal-aid-substantive.herokuapp.com/footer/contact,
we don't want any spacing. Just a new line

Reply:
OK